lib/ansible/module_utils/openstack.py

```diff
@@ -114,16 +114,22 @@ def openstack_cloud_from_module(module, min_version='0.12.0'):
         # Due to the name shadowing we should import other way
         import importlib
         sdk = importlib.import_module('openstack')
+        sdk_version = importlib.import_module('openstack.version')
```

Comment:
Why do you think this additional line is helpful?

Reply:
Did you read the stacktrace mentioned in the description of this PR and in the first commit ?

`sdk.version.__version__` expression raises an `AttributeError` exception when openstacksdk < 0.10.0 is used.

Reply:
> Did you read the stacktrace mentioned in the description of this PR and in the first commit ?

Sure, but I did not understand how this is related, but with your additional exlaination, I understand now.
The commit message could be improved by writing something like
"`openstack.version.__version__` expression raises an AttributeError exception when openstacksdk < 0.10.0 is used. openstack.version is now imported as a module, which works for all openstacksdk versions.

Reply:
PR updated:
- commit message updated
- rebased on devel